ocs_ci/ocs/resources/mcg.py

```diff
@@ -710,3 +710,22 @@ def exec_mcg_cmd(self, cmd, namespace=None, **kwargs):
         result.stdout = result.stdout.decode()
         result.stderr = result.stderr.decode()
         return result
+
+    @property
+    def status(self):
+        """
+        Verify noobaa status output is clean without any errors
+
+        Returns:
+            bool: return False if any of the non optional components of noobaa is not available
+
+        """
+        # Get noobaa status
+        status = self.exec_mcg_cmd('status').stderr
+        for line in status.split('\n'):
+            if 'Not Found' in line:
```

Comment:
I think we can switch to short circuiting for better readability - 
`if 'Not Found' in line and 'Optional' not in line:`
Not a must, though